.travis.yml

```diff
@@ -7,28 +7,18 @@ jobs:
     - language: python
       python: '3.6'
       env:
-        - TEST_TYPE=monitoring_service
-        - SOLC_URL='https://github.com/ethereum/solidity/releases/download/v0.4.23/solc-static-linux'
-        - SOLC_VERSION='v0.4.23'
-
-      before_install:
-        - mkdir -p $HOME/.bin
-        - export PATH=$PATH:$HOME/.bin
-        - ./.travis/download_solc.sh
+        - CODECOV_TOKEN="67d405e6-a7e4-4219-869d-ec6b8ab8c868"
```

Comment:
Do we want to post tokens into public repos?

Reply:
Seems like this isn't needed for public repos. Removed it in 4555483